scout/log/log.py

```diff
@@ -0,0 +1,63 @@
+import os
+import sys
+import logging
+
+def init_log(logger, filename=None, loglevel=None):
```

Comment:
I'm not super sure but why not initialize the root logger in this function? Then all the rest of the loggers will inherit it's formatting and handler, no?

```python
root_logger = logging.getLogger()
```